test/functional-vanilla.bats

```diff
@@ -170,6 +207,12 @@ spec:
   checkApplication
 }
 
+@test "Rename Application" {
+  test iofogctl -v rename application "${APPLICATION_NAME}" "${APPLICATION_NAME}-newname"
+  test iofogctl -v rename application "${APPLICATION_NAME}-newname" "${APPLICATION_NAME}"
```

Comment:
This does not test anything. If `iofogctl rename application` was not doing anything, your test would still pass